daq/panda/main.py

```diff
@@ -40,17 +42,28 @@
   #Read in from the serial. Timeout if nothing is available
   text = ser.readline().decode() #read in one data string
 
+  #optional: Prints out black box data to certain location
+  savePath = 'C:\\Users\\jsand\\OneDrive\\Documents\\SMV\\DAQ\\Trials'
```

Comment:
If you set the path as "os.path.dirname(\_\_file__)"  it will use the current directory as the save path so we won't have to change it when we move the code to the RasPi.
```suggestion
  savePath = os.path.dirname(__file__)
```